taccsite_cms/templates/guides/rsync_data_transfer.html

```diff
@@ -0,0 +1,510 @@
+        </p>
+      </blockquote>
+
+      <p>
+        To transfer <code>my_file.txt</code> from your local computer to your transfer directory (ex. <code>/corral-secure/projects/A2CPS/submissions/utaustin</code>):
+      </p>
+
+      <pre style="max-width: 1000px"><code><u>sftp&gt; </u>put my_file.txt
+<samp>Uploading my_file.txt to <var>/transfer/directory/path</var></samp>
+<samp>my_file.txt     100% ##  #.#          KB/s   ##:#</samp></code></pre>
+
+      <p>
+        To check if <code>my_file.txt</code> is in the <code>utaustin</code>
+        subfolder:
+      </p>
+
+      <pre style="max-width: 500px"><code><u>sftp&gt; </u>ls
+<samp>my_file.txt</samp>
+<samp>utaustin_dir.txt</samp></code></pre>
+
+      <p>
+        To exit out of <code>sftp</code> on the terminal:
+      </p>
+
+      <pre style="max-width: 500px"><code><u>sftp&gt; </u>bye
+<u>localhost1$</u></code></pre>
+
+      <p>
+        If you wish to learn more about <code>sftp</code>, you can do so
+        at <a target="_blank" href="https://man7.org/linux/man-pages/man1/sftp.1.html">the online <code>man</code> page for <code>scp</code></a>.
+      </p>
+
+      <h4 id="rsync_transfer">Transfer Data with <code>rsync</code></h4>
+
+      <p>
+        <code>rsync</code>is a file copying tool that can reduce the amount of data transferred by sending only the differences between the source files on your local system and the existing files in your transfer directory. To transfer a file (ex. <code>my_file.txt</code>) to the remote secure system via <code>rsync</code>, open a terminal on your local computer and navigate to the path where your data file is located.
+      </p>
+      <pre style="max-width: 500px" ><code><u>localhost$ </u>cd Documents/portal-data/</code></pre>
+      <p>
+        Assuming your TACC username is <code>jdoe</code> and you are affiliated with UT Austin, a <code>rsync</code> transfer that pushes
+        <code>my_file.txt</code>
+        from your local computer to the remote secure system would look like this:
+      </p>
+      <pre style="max-width: 1000px"><code><u>localhost$ </u>rsync ./my_file.txt jdoe@<kbd>host</kbd>:<kbd>/transfer/directory/path</kbd></code></pre>
+      <blockquote>
+        <p>If you have not done so already, enter this command in your terminal,
+          replacing the TACC username and your individualized transfer directory name appropriately.</p>
+      </blockquote>
+      <p>
+        If the command returns 0 in your terminal, the data transfer was successful.
+      </p>
+      <p>
+        If you wish to learn more about <code>rsync</code> and how to synchronize your file transfer, you can do so
+        <a target="_blank" href="https://man7.org/linux/man-pages/man1/rsync.1.html">the online <code>man</code> page for <code>rsync</code></a>
+        or follow the file transfer section of the user guide for the relevant storage system:
```

Comment:
SUGGESTION: Refer to it as "the appropriate TACC system" to avoid confusion by overloading the term "storage system" in this context.